doc/sphinx-guides/source/installation/prerequisites.rst

```diff
@@ -104,16 +104,16 @@ You are welcome to experiment with newer versions of PostgreSQL, but please note
 Installing PostgreSQL
 =====================
 
-*For example*, to install PostgreSQL 13 under RHEL7/derivative::
+The application is currently tested on PostgreSQL version 13, though versions 14-16 are supported as of v6.2. To install PostgreSQL 13 under RHEL9/derivative::
 
-	# yum install -y https://download.postgresql.org/pub/repos/yum/reporpms/EL-7-x86_64/pgdg-redhat-repo-latest.noarch.rpm
-	# yum makecache fast
-	# yum install -y postgresql13-server
-	# /usr/pgsql-13/bin/postgresql-13-setup initdb
-	# /usr/bin/systemctl start postgresql-13
-	# /usr/bin/systemctl enable postgresql-13
+	# sudo dnf install -y https://download.postgresql.org/pub/repos/yum/reporpms/EL-9-x86_64/pgdg-redhat-repo-latest.noarch.rpm
+	# sudo dnf check-update
+	# sudo dnf install -y postgresql13-server
+	# sudo /usr/pgsql-13/bin/postgresql-13-setup initdb
+	# sudo /usr/bin/systemctl start postgresql-13
+	# sudo /usr/bin/systemctl enable postgresql-13
```

Comment:
I think the `#` signals that these commands are run as root, meaning `sudo` isn't needed. It could help to more explicitly say the commands need to be run as root.